.eslintignore

```diff
@@ -0,0 +1 @@
+*.js
```

Comment:
Which files are these? I think there's only one, `src/js/jwt-decode.js` ... i think it would be better to explicitly exclude that file ... that way, if other `.js` files are added in the future, warnings will show and a decision can be made as to whether those files should be excluded. wdyt?

Reply:
There are a few other config files in JS such as:
`jest.config.js`, `postcss.config.js`, and `webpack.config.js`

I do agree however that it would be more helpful to see the warnings of possible future `.js` files rather than ignoring all of them.